pallets/liquidity-pools/src/tests.rs

```diff
@@ -213,13 +213,21 @@ mod transfer_tranche_tokens {
 		DomainAddressToAccountId::mock_convert(|_| CONTRACT_ACCOUNT_ID);
 		Time::mock_now(|| NOW);
 		Permissions::mock_has(move |scope, who, role| {
-			assert_eq!(who, CONTRACT_ACCOUNT_ID);
 			assert!(matches!(scope, PermissionScope::Pool(POOL_ID)));
-			assert!(matches!(
-				role,
-				Role::PoolRole(PoolRole::TrancheInvestor(TRANCHE_ID, NOW_SECS))
-			));
-			true
+			assert_eq!(who, CONTRACT_ACCOUNT_ID);
+			match role {
```

Comment:
NIT. Could we add new test in the `erroring_out` section checking: the error `InvestorDomainAddressFrozen`

Reply:
[a5f2e6e](https://github.com/centrifuge/centrifuge-chain/pull/1947/commits/a5f2e6e272c2b8400190994a4529f97ca012ad46)